client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -37,8 +38,9 @@ function PostActionsEllipsisMenuDuplicate( {
 	}
 
 	return (
-		<PopoverMenuItem href={ duplicateUrl } onClick={ onDuplicateClick } icon="pages">
-			{ translate( 'Duplicate', { context: 'verb' } ) }
+		<PopoverMenuItem href={ duplicateUrl } onClick={ onDuplicateClick }>
```

Comment:
Minor: PopoverMenuItem does have a convenience `icon` prop that renders the gridicon. So alternatively we might write:

`<PopoverMenuItem href={ duplicateUrl } onClick={ onDuplicateClick } icon="clipboard">`

Let me know if you're interested in updating! I'll accept the PR in either case.

Reply:
Thanks @gwwar! I've switched that now.